server/views/pages/vary/timeline.njk

```diff
@@ -0,0 +1,120 @@
+{% extends "layout.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/tabs/macro.njk" import govukTabs %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from "partials/varyCallToActions.njk" import cta %}
+{% from "moj/components/timeline/macro.njk" import mojTimeline %}
+
+{% set pageTitle = applicationName + " - Vary a licence - licence history" %}
+{% set pageId = "view-timeline-page" %}
+{% set backLinkHref = "/licence/vary/caseload" %}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            <h1 class="govuk-heading-l">Current licence and licence history</h1>
+        </div>
+    </div>
+
+    {%- set currentLicenceHtml  %}
+        <form method="POST">
+            <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+            {{ cta(callToActions, licence) }}
+            {{ currentLicenceEvents(timelineEvents) }}
+        </form>
+    {% endset -%}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            {{ govukTabs({
+                items: [
+                    {
+                        label: "Current licence",
+                        id: "current-licence",
+                        panel: {
+                            html: currentLicenceHtml
+                        }
+                    }
+                ]
+            }) }}
+        </div>
+    </div>
+
+{% endblock %}
+
+{% macro currentLicenceEvents(events) %}
+    <div class="moj-timeline">
+        {% for event in events %}
+            <div class="moj-timeline__item">
+                <h2 class="moj-timeline__title">{{ event.title }}</h2>
+                <p class="moj-timeline__byline">by {{ event.createdBy | title }}</p>
+                <p class="moj-timeline__date">last update: {{ event.lastUpdate | datetimeToDate }} at {{ event.lastUpdate | datetimeTo12HourTime }}</p>
```

Comment:
Think these should be `<span>`s for the screen readers

Reply:
Which bits?

Reply:
The two `<p>` tags